### ISSUE TITLE
Error when submitting store metadata

### ISSUE THREAD

**Issue by shizu2014** [fastlane/fastlane#16928]
<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                          |
| --------------------------- | -------------------------------------------------------------- |
| OS                          | 10.15.5                                                        |
| Ruby                        | 2.6.3                                                          |
| Bundler?                    | false                                                          |
| Git                         | git version 2.21.0 (Apple Git-122)                             |
| Installation Source         | /usr/local/bin/fastlane                                        |
| Host                        | Mac OS X 10.15.5 (19F101)                                      |
| Ruby Lib Dir                | /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib |
| OpenSSL Version             | LibreSSL 2.8.3                                                 |
| Is contained                | false                                                          |
| Is homebrew                 | false                                                          |
| Is installed via Fabric.app | false                                                          |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                    |
| Xcode Version               | 11.1                                                           |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | zh_CN.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
# This file contains the fastlane.tools configuration
# You can find the documentation at https://docs.fastlane.tools
#
# For a list of all available actions, check out
#
#     https://docs.fastlane.tools/actions
#
# For a list of all available plugins, check out
#
#     https://docs.fastlane.tools/plugins/available-plugins
#

# Uncomment the line if you want fastlane to automatically update itself
# update_fastlane

default_platform(:ios)

platform :ios do
  desc "Push a new release build to the App Store"
  lane :release do
    build_app(scheme: "xxx")
    upload_to_app_store
  end
end

```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
app_identifier("xxx") # The bundle identifier of your app
apple_id("xxx") # Your Apple email address

itc_team_id("xxx") # App Store Connect Team ID
team_id("xxx") # Developer Portal Team ID

# For more information about the Appfile, see:
#     https://docs.fastlane.tools/advanced/#appfile

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.153.0 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.3.0        |
| slack-notifier            | 2.3.2        |
| atomos                    | 0.1.3        |
| CFPropertyList            | 2.3.6        |
| claide                    | 1.0.3        |
| colored2                  | 3.1.2        |
| nanaimo                   | 0.2.6        |
| xcodeproj                 | 1.16.0       |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.3.0        |
| terminal-notifier         | 2.0.0        |
| unicode-display_width     | 1.7.0        |
| terminal-table            | 1.8.0        |
| plist                     | 3.5.0        |
| public_suffix             | 2.0.5        |
| addressable               | 2.7.0        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| tty-screen                | 0.7.1        |
| tty-cursor                | 0.7.1        |
| tty-spinner               | 0.9.3        |
| babosa                    | 1.0.3        |
| colored                   | 1.2          |
| highline                  | 1.7.10       |
| commander-fastlane        | 4.4.6        |
| excon                     | 0.73.0       |
| unf_ext                   | 0.0.7.7      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20190701 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| faraday                   | 1.0.1        |
| faraday_middleware        | 1.0.0        |
| fastimage                 | 2.1.7        |
| gh_inspector              | 1.1.3        |
| json                      | 2.1.0        |
| mini_magick               | 4.10.1       |
| rubyzip                   | 2.3.0        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 1.0.0        |
| dotenv                    | 2.7.5        |
| bundler                   | 1.17.2       |
| naturally                 | 2.2.0        |
| simctl                    | 1.6.8        |
| jwt                       | 2.1.0        |
| uber                      | 0.1.0        |
| declarative               | 0.0.10       |
| declarative-option        | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 3.1.2        |
| mini_mime                 | 1.0.2        |
| multi_json                | 1.14.1       |
| signet                    | 0.14.0       |
| memoist                   | 0.16.2       |
| os                        | 1.1.0        |
| googleauth                | 0.12.0       |
| httpclient                | 2.8.3        |
| google-api-client         | 0.38.0       |
| google-cloud-env          | 1.3.1        |
| google-cloud-errors       | 1.0.0        |
| google-cloud-core         | 1.5.0        |
| digest-crc                | 0.5.1        |
| google-cloud-storage      | 1.26.1       |
| emoji_regex               | 1.0.1        |
| jmespath                  | 1.4.0        |
| aws-partitions            | 1.313.0      |
| aws-eventstream           | 1.1.0        |
| aws-sigv4                 | 1.1.3        |
| aws-sdk-core              | 3.95.0       |
| aws-sdk-kms               | 1.31.0       |
| aws-sdk-s3                | 1.64.0       |
| forwardable               | 1.2.0        |
| logger                    | 1.3.0        |
| stringio                  | 0.0.2        |
| ipaddr                    | 1.2.2        |
| openssl                   | 2.1.2        |
| ostruct                   | 0.1.0        |
| strscan                   | 1.0.0        |
| date                      | 2.0.0        |
| fileutils                 | 1.1.0        |
| etc                       | 1.0.1        |
| io-console                | 0.4.7        |
| zlib                      | 1.0.0        |
| libxml-ruby               | 3.1.0        |
| rexml                     | 3.1.9        |
| psych                     | 3.1.0        |
| mutex_m                   | 0.1.0        |
| webrick                   | 1.4.2        |
</details>

*generated on:* **2020-07-24**
</details>

Deliverfile
app_review_information({
  
  demo_account_required:false,
  contact_first_name: "xxx",
  contact_last_name: "xxx",
  contact_phone: "+xxx",
  contact_email: "xxxx.com",
  demo_account_name: "",
  demo_account_password: "",
  notes: ""
})

**Comment by shizu2014** [comment]
@joshdholtz 
pi/client.rb:163:in `handle_response': [!] The provided entity includes an unknown attribute - 'demo_account_required' is not an attribute on the resource 'appStoreReviewDetails' (Spaceship::UnexpectedResponse)

**Comment by joshdholtz** [comment]
@shizu2014 Thank you for creating a new issue! Are you able to post your entire _fastlane_ output (from beginning to end)? It gives me a lot more info to diagnose the issue.

**Comment by shizu2014** [comment]
[23:43:33]: Successfully exported and compressed dSYM file
[23:43:33]: Successfully exported and signed the ipa file:
[23:43:33]: /Users/xxx/Documents/xxx/0724/xxx/xx.ipa
[23:43:33]: ---------------------------------
[23:43:33]: --- Step: upload_to_app_store ---
[23:43:33]: ---------------------------------
[23:43:33]: Successfully loaded '/Users/xx/xx/xx/xx/xx/fastlane/Deliverfile' 📄

+----------------------------+--------------------------------------+
|           Detected Values from './fastlane/Deliverfile'           |
+----------------------------+--------------------------------------+
| price_tier                 | 0                                    |
| app_review_information     | {:demo_account_required=>false,      |
|                            | :contact_first_name=>"xxx",           |
|                            | :contact_last_name=>"xxx",       |
|                            | :contact_phone=>"+xxx",    |
|                            | :contact_email=>"xxxx@yandex.co  |
|                            | m", :demo_account_name=>"",          |
|                            | :demo_account_password=>"",          |
|                            | :notes=>""}                          |
| submission_information     | {:add_id_info_uses_idfa=>false,      |
|                            | :export_compliance_uses_encryption=  |
|                            | >false,                              |
|                            | :export_compliance_encryption_updat  |
|                            | ed=>false,                           |
|                            | :content_rights_contains_third_part  |
|                            | y_content=>false}                    |
| primary_category           | Entertainment                        |
| skip_screenshots           | false                                |
| skip_metadata              | false                                |
| force                      | true                                 |
| run_precheck_before_submit | false                                |
| copyright                  | Copyright 2020 xxx    |
| automatic_release          | true                                 |
| submit_for_review          | true                                 |
| app_rating_config_path     | ./rating.json                        |
+----------------------------+--------------------------------------+

[23:43:33]: Login to App Store Connect (vjthnh@163.com)
[23:43:39]: Login successful

+--------------------------------------+--------------------------------------+
|                           deliver 2.153.0 Summary                           |
+--------------------------------------+--------------------------------------+
| ipa                                  | /Users/xxx/Documents/workSpace/  |
|                                      | xxxxx/xxxx/xxxx.ipa       |
| screenshots_path                     | ./fastlane/screenshots               |
| metadata_path                        | ./fastlane/metadata                  |
| app_version                          | 1.0                                  |
| username                             | xxxx@163.com                       |
| app_identifier                       |xxxx                |
| platform                             | ios                                  |
| edit_live                            | false                                |
| use_live_version                     | false                                |
| skip_binary_upload                   | false                                |
| skip_screenshots                     | false                                |
| skip_metadata                        | false                                |
| skip_app_version_update              | false                                |
| force                                | true                                 |
| overwrite_screenshots                | false                                |
| submit_for_review                    | true                                 |
| reject_if_possible                   | false                                |
| automatic_release                    | true                                 |
| phased_release                       | false                                |
| reset_ratings                        | false                                |
| price_tier                           | 0                                    |
| app_rating_config_path               | ./rating.json                        |
| submission_information.add_id_info_  | false                                |
| uses_idfa                            |                                      |
| submission_information.export_compl  | false                                |
| iance_uses_encryption                |                                      |
| submission_information.export_compl  | false                                |
| iance_encryption_updated             |                                      |
| submission_information.content_righ  | false                                |
| ts_contains_third_party_content      |                                      |
| team_id                              | xxx                            |
| dev_portal_team_id                   | xxx                           |
| run_precheck_before_submit           | false                                |
| precheck_default_rule_level          | warn                                 |
| copyright                            | Copyright 2020 xxxx    |
| primary_category                     | Entertainment                        |
| app_review_information.demo_account  | false                                |
| _required                            |                                      |
| app_review_information.contact_firs  | bi                                   |
| t_name                               |                                      |
| app_review_information.contact_last  | xxx                              |
| _name                                |                                      |
| app_review_information.contact_phon  | +xxxx                       |
| e                                    |                                      |
| app_review_information.contact_emai  | xxx@yandex.com                  |
| l                                    |                                      |
| ignore_language_directory_validatio  | false                                |
| n                                    |                                      |
| precheck_include_in_app_purchases    | true                                 |
+--------------------------------------+--------------------------------------+

[23:43:41]: Making sure the latest version on App Store Connect matches '1.0'...
[23:43:43]: '1.0' is the latest version on App Store Connect
[23:43:46]: Loading './fastlane/metadata/zh-Hans/description.txt'...
[23:43:46]: Loading './fastlane/metadata/zh-Hans/keywords.txt'...
[23:43:46]: Loading './fastlane/metadata/zh-Hans/release_notes.txt'...
[23:43:46]: Loading './fastlane/metadata/zh-Hans/support_url.txt'...
[23:43:46]: Loading './fastlane/metadata/zh-Hans/marketing_url.txt'...
[23:43:46]: Loading './fastlane/metadata/zh-Hans/promotional_text.txt'...
[23:43:46]: Loading './fastlane/metadata/zh-Hans/name.txt'...
[23:43:46]: Loading './fastlane/metadata/zh-Hans/subtitle.txt'...
[23:43:46]: Loading './fastlane/metadata/zh-Hans/privacy_url.txt'...
[23:43:46]: Loading './fastlane/metadata/zh-Hans/apple_tv_privacy_policy.txt'...
[23:43:46]: Loading './fastlane/metadata/copyright.txt'...
[23:43:46]: Loading './fastlane/metadata/primary_category.txt'...
[23:43:46]: Loading './fastlane/metadata/secondary_category.txt'...
[23:43:46]: Loading './fastlane/metadata/primary_first_sub_category.txt'...
[23:43:46]: Loading './fastlane/metadata/primary_second_sub_category.txt'...
[23:43:46]: Loading './fastlane/metadata/secondary_first_sub_category.txt'...
[23:43:46]: Loading './fastlane/metadata/secondary_second_sub_category.txt'...
[23:43:56]: Will begin uploading metadata for '1.0' on App Store Connect
[23:43:56]: Skipping 'release_notes'... this is the first version of the app
[23:43:56]: Uploading metadata to App Store Connect for version
[23:43:57]: Uploading metadata to App Store Connect for localized version 'zh-Hans'
[23:43:58]: Uploading metadata to App Store Connect for localized info 'zh-Hans'
[23:44:02]: Category 'Entertainment' from iTunesConnect has been deprecated. Please replace with 'ENTERTAINMENT'
[23:44:02]: You can find more info at https://docs.fastlane.tools/actions/deliver/#reference
[23:44:09]: Uploading app review information to App Store Connect
[23:44:10]: Error fetching app store review detail - The specified resource does not exist - There is no resource of type 'appStoreReviewDetails' with id 'null'
+--------------------+--------------------------------------+
|                       Lane Context                        |
+--------------------+--------------------------------------+
| DEFAULT_PLATFORM   | ios                                  |
| PLATFORM_NAME      | ios                                  |
| LANE_NAME          | ios release                          |
| IPA_OUTPUT_PATH    | /Users/Mac2020/Documents/workSpace/  |
|                    | 0724/soccerBall/soccerBall.ipa       |
| XCODEBUILD_ARCHIVE | /Users/Mac2020/Library/Developer/Xc  |
|                    | ode/Archives/2020-07-24/soccerBall   |
|                    | 2020-07-24 23.43.09.xcarchive        |
| DSYM_OUTPUT_PATH   | /Users/Mac2020/Documents/workSpace/  |
|                    | 0724/soccerBall/soccerBall.app.dSYM  |
|                    | .zip                                 |
+--------------------+--------------------------------------+
[23:44:10]: The provided entity includes an unknown attribute - 'demo_account_required' is not an attribute on the resource 'appStoreReviewDetails'

+------+---------------------+-------------+
|             fastlane summary             |
+------+---------------------+-------------+
| Step | Action              | Time (in s) |
+------+---------------------+-------------+
| 1    | default_platform    | 0           |
| 2    | build_app           | 26          |
| 💥   | upload_to_app_store | 37          |
+------+---------------------+-------------+

[23:44:10]: fastlane finished with errors

Looking for related GitHub issues on fastlane/fastlane...

Traceback (most recent call last):
	34: from /usr/local/bin/fastlane:23:in `<main>'
	33: from /usr/local/bin/fastlane:23:in `load'
	32: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/bin/fastlane:23:in `<top (required)>'
	31: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/fastlane/lib/fastlane/cli_tools_distributor.rb:119:in `take_off'
	30: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/fastlane/lib/fastlane/commands_generator.rb:41:in `start'
	29: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/fastlane/lib/fastlane/commands_generator.rb:352:in `run'
	28: from /Library/Ruby/Gems/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/delegates.rb:15:in `run!'
	27: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:76:in `run!'
	26: from /Library/Ruby/Gems/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/runner.rb:476:in `run_active_command'
	25: from /Library/Ruby/Gems/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:153:in `run'
	24: from /Library/Ruby/Gems/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:178:in `call'
	23: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/fastlane/lib/fastlane/commands_generator.rb:108:in `block (2 levels) in run'
	22: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/fastlane/lib/fastlane/command_line_handler.rb:36:in `handle'
	21: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/fastlane/lib/fastlane/lane_manager.rb:47:in `cruise_lane'
	20: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/fastlane/lib/fastlane/runner.rb:45:in `execute'
	19: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/fastlane/lib/fastlane/runner.rb:45:in `chdir'
	18: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
	17: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/fastlane/lib/fastlane/lane.rb:33:in `call'
	16: from Fastfile:22:in `block (2 levels) in parsing_binding'
	15: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/fastlane/lib/fastlane/fast_file.rb:159:in `method_missing'
	14: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/fastlane/lib/fastlane/runner.rb:157:in `trigger_action_by_name'
	13: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/fastlane/lib/fastlane/runner.rb:229:in `execute_action'
	12: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/fastlane/lib/fastlane/runner.rb:229:in `chdir'
	11: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/fastlane/lib/fastlane/runner.rb:255:in `block in execute_action'
	10: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/fastlane/lib/fastlane/actions/actions_helper.rb:50:in `execute_action'
	 9: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/fastlane/lib/fastlane/runner.rb:263:in `block (2 levels) in execute_action'
	 8: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/fastlane/lib/fastlane/actions/upload_to_app_store.rb:17:in `run'
	 7: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/deliver/lib/deliver/runner.rb:42:in `run'
	 6: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/deliver/lib/deliver/runner.rb:122:in `upload_metadata'
	 5: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/deliver/lib/deliver/upload_metadata.rb:324:in `upload'
	 4: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/deliver/lib/deliver/upload_metadata.rb:565:in `set_review_information'
	 3: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/spaceship/lib/spaceship/connect_api/models/app_store_version.rb:122:in `create_app_store_review_detail'
	 2: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/spaceship/lib/spaceship/connect_api/tunes/tunes.rb:607:in `post_app_store_review_detail'
	 1: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/spaceship/lib/spaceship/connect_api/client.rb:99:in `post'
/Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.0/spaceship/lib/spaceship/connect_api/client.rb:163:in `handle_response': [!] The provided entity includes an unknown attribute - 'demo_account_required' is not an attribute on the resource 'appStoreReviewDetails' (Spaceship::UnexpectedResponse)

**Comment by shizu2014** [comment]
The previous app_review_information should be filled out like this:
app_review_information({
  first_name: " xxx",
  last_name: "xxx",
  phone_number: "+xxxx",
  email_address: "xxxx@hotmail.com",
  demo_user: "",
  demo_password: "",
  notes: ""
})
But if you fill in this way and report more errors.

**Comment by markmuniz** [comment]
I'm getting the same error after updating to `2.153.0`.

**Comment by alex-benoit** [comment]
`upload_to_testflight` hangs as well once build has been uploaded and when setting changelog metadata

**Comment by joshdholtz** [comment]
@shizu2014 @markmuniz Which version were you both coming from?

**Comment by joshdholtz** [comment]
@alex-benoit `upload_to_testflight` didn’t have any changes in this latest version 🤔 Would you be able to open an issue with what you are seeing? I think yours might be something different

**Comment by markmuniz** [comment]
> @shizu2014 @markmuniz Which version were you both coming from?

`2.152.0`

**Comment by joshdholtz** [comment]
Ahhh, I see the issue! I will have a new version (`2.153.1`) out in about an hour or so.

**Comment by joshdholtz** [comment]
This is fixed in `2.153.1` now!

**Comment by alex-benoit** [comment]
@joshdholtz Indeed - this was my issue https://github.com/fastlane/fastlane/issues/16932
Also related to metadata uploading but completely unrelated...
